redis_store.go

```diff
@@ -178,7 +178,7 @@ func (this *RedisStore) Poll(c chan []byte, queue string) error {
 				continue
 			}
 
-			message, err := redis.Bytes(consumer.Do("LPOP", queue))
+			message, err := redis.Bytes(consumer.Do("BLPOP", queue, 10))
```

Comment:
@samuelebistoletti Could you put this in a different PR? Also this change is related to https://github.com/Imgur/incus/issues/53